be/src/exec/vectorized/hdfs_scanner_parquet.cpp

```diff
@@ -40,6 +43,7 @@ void HdfsParquetScanner::do_update_counter(HdfsScanProfile* profile) {
     group_dict_filter_timer = ADD_CHILD_TIMER(root, "GroupDictFilter", kParquetProfileSectionPrefix);
     group_dict_decode_timer = ADD_CHILD_TIMER(root, "GroupDictDecode", kParquetProfileSectionPrefix);
 
+    COUNTER_UPDATE(parsed_bytes_read, _stats.page_bytes_read);
```

Comment:
why use different name, it's confusing. a more accurate name plz

Reply:
do you have any suggestion?